### DISCUSSION TITLE
uptime dashboard for HA

### DISCUSSION THREAD

**Discussion by mariomare22** [indomus/forum#512] [HUB: Home Assistant]
Salve a tutti,

Mi piacerebbe creare una dashboard di questo genere ([Link](https://github.com/louislam/uptime-kuma)) per monitorare alcuni portali web che mantengo.
![image](https://user-images.githubusercontent.com/12135860/125463415-6bb06641-3b56-47d4-9f29-228330d75891.png)

Quale e' la migliore strategia per farlo?

**Comment by maxcanna** [comment]
Personalmente credo che una dashboard del genere esuli (molto) dalle responsabilità di HA

**Reply by KC-inDomus** [reply]
concordo, non è il suo mestiere

**Comment by mariomare22** [comment]
ok ricevuto! Grazie per la risposta.
